tools/mnemonic_generator.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# A simple/silly tool for generating a mnemonic with known words. Possibly useful for testing.
+# Usage:
+#  $ python tools/mnemonic_generator.py exotic toilet dance
+#  exotic toilet dance <rest of mnemonic>
+
+import argparse
+import sys
+
+from bitstring import BitArray
+from chia.util.keychain import bip39_word_list, bytes_to_mnemonic
+from secrets import token_bytes
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-l", "--list", action="store_true")
```

Comment:
This is more of a subcommand situation since it just does something entirely different.